### DISCUSSION TITLE
Contributing?

### DISCUSSION THREAD

**Discussion by geekpulp** [mastodon/mastodon#26377] [Q&A] [answered]
Hi team, I’d like to contribute to the project. Is there a “how to contribute” document somewhere? I can’t seem to find one.

**Comment by geekpulp** [comment]
If there isn’t maybe my first contribution could be to make one?

**Reply by wteo** [reply, marked as answer]
Please check the link below. 

https://github.com/mastodon/mastodon/blob/main/CONTRIBUTING.md
